hologram_dashboard/server/request_handler/request_handler.cc

```diff
@@ -35,55 +25,46 @@ void RequestHandler::GetLastRefreshed(const Pistache::Rest::Request& request,
   response.send(Pistache::Http::Code::Ok, "Wed May 19 15:46:11 2020");
 }
 
+nlohmann::json RequestHandler::ConvertProtoIntoJSON(
+    const std::vector<HologramDataAvailability>& hologram_data_availabilities) {
+  nlohmann::json message = nlohmann::json::array();
+  int index = 0;
+  for (const HologramDataAvailability& hologram_data_availability :
+      hologram_data_availabilities) {
+    message.push_back(nlohmann::json::object());
+    message[index]["sourceType"] = SourceType_Name(hologram_data_availability.source_type());
+    message[index]["availabilityStatusList"] = nlohmann::json::array();
+    for (const HologramDataAvailability_AvailabilityStatus& 
+        availability_status : hologram_data_availability.availability_status()) 
+    {
+      message[index]["availabilityStatusList"].push_back({
+          {"date", availability_status.date()}, 
+          {"isAvailable", availability_status.source_ingestion_status() == 
+          HologramDataAvailability::INGESTED}});
+    }
+    index++;
+  }
```

Comment:
As discussed offline, revert the server side change about returned JSON object? Thanks!

Reply:
reverted